tsconfig.json

```
@@ -3,15 +3,16 @@
   "display": "Node LTS",
   "_version": "20.1.0",
   "compilerOptions": {
+    "target": "ESNext",
+    "module": "Node16",
```

```suggestion
    "target": "esnext",
    "module": "node16",
```

mấy cái này viết thường, tích hợp đc nhiều tools hơn. Có vài tool k hiểu option này viết hoa, ngoài thằng TS Compiler